tests/test_unit.py

```diff
@@ -115,6 +115,41 @@ def test_normalize(self):
             )
             df_norm = df_utils.normalize(df, data_params)
 
+    def test_add_lagged_regressors(self):
+        NROWS = 512
+        EPOCHS = 3
+        BATCH_SIZE = 32
+        
+        df = pd.read_csv(PEYTON_FILE, nrows=NROWS)
+
+        df["A"] = df["y"].rolling(7, min_periods=1).mean()
+        df["B"] = df["y"].rolling(15, min_periods=1).mean()
+        df["C"] = df["y"].rolling(30, min_periods=1).mean()
+        
+        check = {
+            "1": ["A"],
+            "2": ["A", "B"],
```

Comment:
instead of checking for two, add a check where there is just one, but no list -> just "A"